packages/plugin-rsc/playwright.config.ts

```diff
@@ -27,7 +27,8 @@ export default defineConfig({
       use: devices['Desktop Safari'],
     },
   ],
-  workers: 2,
+  // workers: 2,
+  workers: 1,
```

Comment:
(on my local system, removing config really speeds up e2E)

```suggestion
  workers: 2,
```